README.md

```diff
@@ -82,6 +82,7 @@ The above options are best for multi-page sites. Single-page apps have a few opt
 * `priority`: Boolean specifying preferred priority for fetches. Defaults to `false`. `true` will attempt to use the `fetch()` API where supported (rather than rel=prefetch)
 * `origins`: Static array of URL hostname strings that are allowed to be prefetched. Defaults to the same domain origin, which prevents _any_ cross-origin requests.
 * `ignores`: A RegExp, Function, or Array that further determines if a URL should be prefetched. These execute _after_ origin matching.
+* `limit`: Limits the number of links to be prefetched. Defaults to `Infinity`.
```

Comment:
I think we'll want to provide more detail on `limit`. Folks may be wondering if this is the total limit of links quicklink will ever prefetch in a session or if it's the number of links that will be prefetched at a time. Maybe we can make that more clear?